docker/release/cudaq.ext.Dockerfile

```diff
@@ -20,10 +20,13 @@ RUN for folder in `find "$CUDA_QUANTUM_PATH/assets"/*$(uname -m)/* -maxdepth 0 -
     && rm "$CUDA_QUANTUM_PATH/bin/migrate_assets.sh"
 
 # Install additional runtime dependencies.
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        cuda-nvtx-11-8 libcusolver-dev-11-8 libopenblas-openmp-dev \
+RUN apt-get install -y --no-install-recommends \
+        cuda-nvtx-11-8 libcusolver-11-8 libopenblas-openmp-dev \
         # just here for convenience:
-        curl jq \
-    && apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/*
+        curl jq
+
+# Make sure that apt-get remains updated at the end!;
+# If we don't do that, then apt-get will get confused when some CUDA
+# components are already installed but not all of them.
```

Comment:
Is this supposed to contain the `RUN apt-get update` here?

Reply:
That shouldn't be necessary, since the base image ran apt-get update and we don't add any additional sources. Hence I also removed the apt-get update prior to installing the additional cuda packages.

Reply:
> That shouldn't be necessary, since the base image ran apt-get update and we don't add any additional sources.

OK, I'm not sure I understand why the new comment is necessary then. It makes it seem like something is missing. If I'm understanding correctly, perhaps it could be updated to say:
```suggestion
# Make sure that apt-get remains updated at the end.
# If you add any additional sources at this point, be sure to add
# `RUN apt-get update` because if we don't do that, then apt-get
# will get confused when some CUDA components are already
# installed but not all of them.
```